xdsl/dialects/arith.py

```diff
@@ -765,13 +768,16 @@ def parse(cls, parser: Parser):
         operand1 = parser.parse_unresolved_operand()
         parser.parse_punctuation(",")
         operand2 = parser.parse_unresolved_operand()
+        fastmath = FastMathFlagsAttr("none")
+        if parser.parse_optional_keyword("fastmath") is not None:
+            fastmath = FastMathFlagsAttr(FastMathFlagsAttr.parse_parameter(parser))
```

Comment:
This is parsing the old(/new mlir) version right?

Reply:
I actually have no idea how to answer this. [This page](https://mlir.llvm.org/docs/Dialects/ArithOps/#arithcmpf-arithcmpfop) seems to show it in the `'fastmath<fast>` format separate from the property dict, but it looks like the attribute dict is also used in some other places in `xdsl`. How can I tell which version should be supported, and see the documentation for that version?

Reply:
That page is the documentation for mlir master. I believe the version of mlir we have current compatibility with is 98e674c9f16d677d95c67bc130e267fae331e43c , though the link to this seems to have gone missing from the readme at some point